README.md

```diff
@@ -360,7 +360,7 @@ def extwis():
 
     # Set the system and user URLs
     system_url = "https://raw.githubusercontent.com/danielmiessler/fabric/main/patterns/extract_wisdom/system.md"
-    user_url = "https://raw.githubusercontent.com/danielmiessler/fabric/main/patterns/extract_wisdom/user.md"
+    user_url = "https://github.com/danielmiessler/fabric/blob/main/patterns/extract_wisdom/README.md"
```

Comment:
This should be https://raw.githubusercontent.com/danielmiessler/fabric/main/patterns/extract_wisdom/README.md (the raw github file)

Reply:
Thank you! Corrected.